src/main/java/org/datavyu/plugins/examples/SimpleAVFoundationMediaPlayer.java

```diff
@@ -0,0 +1,28 @@
+package org.datavyu.plugins.examples;
+
+import org.datavyu.plugins.nativeosx.AVFoundationMediaPlayer;
+import org.datavyu.plugins.MediaPlayer;
+
+import javax.swing.*;
+import java.io.File;
+import java.net.URI;
+
+public class SimpleAVFoundationMediaPlayer {
+    public static void main(String[] args) {
+        // Define the media file
+        URI mediaPath = new File("Nature_30fps_1080p.mp4").toURI();
+
+        // Create the media player using the constructor with File
+        MediaPlayer mediaPlayer = new AVFoundationMediaPlayer(mediaPath, new JDialog());
+
+        mediaPlayer.addMediaErrorListener(
+            (source, errorCode, message)
+                -> System.err.println(errorCode + ": " + message));
```

Comment:
Maybe add the source to the message, e.g. source.getClass

Reply:
Done!